licenses/inlong-manager/LICENSE

```diff
@@ -587,7 +587,7 @@ The text of each license is the standard Apache 2.0 license.
   org.apache.twill:twill-discovery-api:0.6.0-incubating - Apache Twill discovery service API (https://twill.apache.org/), (The Apache Software License, Version 2.0)
   org.apache.twill:twill-discovery-core:0.6.0-incubating - Apache Twill discovery service implementations (https://twill.apache.org/), (The Apache Software License, Version 2.0)
   org.apache.twill:twill-zookeeper:0.6.0-incubating - Apache Twill ZooKeeper client library (https://twill.apache.org/), (The Apache Software License, Version 2.0)
-  com.fasterxml.woodstox:woodstox-core:5.0.3 - Woodstox (https://github.com/FasterXML/woodstox/tree/woodstox-core-5.0.3), (The Apache License, Version 2.0)
```

Comment:
I guess change version in sort-connectors cannot change the version in manager module